File: Replace all vowels in the String with the * Character [Private][UnPublished] - GFG/replace-all-vowels-in-the-string-with-the-character-private-un-published.cpp
// { Driver Code Starts
// Initial Template for C++
#include <bits/stdc++.h>
using namespace std;

 // } Driver Code Ends
// User function Template for C++
class GFG {

  public:
    string replaceVowels(string st) {
           string vowels = "aeiouAEIOU";
    for(int i=0;i<st.length();i++){
        if(vowels.find(st[i])!= string::npos){
            st[i] = '*';
        }
    }
    return st;
    }
};

// { Driver Code Starts.

int main() {
    int t;
    cin >> t;
    while (t--) {
        string st;

        cin >> st;

        GFG ob;

        cout << ob.replaceVowels(st) << endl;
    }
    return 0;
};  // } Driver Code Ends

File: Replace all vowels in the String with the * Character [Private][UnPublished] - GFG/replace-all-vowels-in-the-string-with-the-character-private-un-published.py
#User function Template for python3

def replaceVowels(s):
     vowels = ['a', 'e', 'i', 'o', 'u','A', 'E', 'I', 'O', 'U']
     for i in range(len(s)):
        if s[i] in vowels:
            s = s[:i] + '*' + s[i+1:]
     return s


#{ 
#  Driver Code Starts
#Initial Template for Python 3

if __name__ == "__main__":
    t = int(input())
    while(t>0):
        s = input()
        print(replaceVowels(s))
        t = t-1


# } Driver Code Ends